Xamarin.Forms.Platform.Android/Material/MaterialActivityIndicatorRenderer.cs

```diff
@@ -136,32 +135,32 @@ public override bool OnTouchEvent(MotionEvent e)
 			return _motionEventHelper.HandleMotionEvent(Parent, e);
 		}
 
-		void UpdateColors()
+		void UpdateColorsAndRuning()
 		{
 			if (Element == null || Control == null)
 				return;
 
-			// TODO: BackgroundColor is not supported by this control.
+			var background = Element.BackgroundColor.IsDefault 
+				? AColor.Transparent 
+				: Element.BackgroundColor.ToAndroid();
+			(_control.Background as GradientDrawable)?.SetColor(background);
 
-			Color progressColor = Element.Color;
-
-			if (progressColor.IsDefault)
+			if (Element.IsRunning)
 			{
-				var progress = MaterialColors.Light.PrimaryColor;
+				var progress = Element.Color.IsDefault 
+					? MaterialColors.Light.PrimaryColor 
+					: Element.Color.ToAndroid();
 				_control.IndeterminateTintList = ColorStateList.ValueOf(progress);
 			}
 			else
 			{
-				var progress = progressColor.ToAndroid();
-				_control.IndeterminateTintList = ColorStateList.ValueOf(progress);
+				_control.Visibility = Element.BackgroundColor.IsDefault 
+					? ViewStates.Gone 
+					: ViewStates.Visible;
```

Comment:
Just checking @PureWeen, is this something we want? If I am understanding correctly, the track will remain visible if a color is set, but not running. I think this is the correct action, but I believe it differs from the existing implementations. We are doing new controls, so we can do what we want here.

Reply:
This need for show background circle if backgroud color is set.